guides/v2.0/config-guide/bootstrap/mage-dirs.md

```diff
@@ -18,11 +18,11 @@ The `MAGE_DIRS` environment variable enables you to specify custom base director
 
 ## Set MAGE\_DIRS   {#dirs-set}
 
-Specify an associative array where keys are constants from <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Filesystem/DirectoryList.php" target="_blank">\Magento\App\Filesystem\DirectoryList</a> and values are absolute paths of directories or their {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} paths, respectively.
+Specify an associative array where keys are constants from [\\Magento\\App\\Filesystem\\DirectoryList]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Filesystem/DirectoryList.php){: target="_blank"} and values are absolute paths of directories or their {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} paths, respectively.
```

Comment:
I don't think there should be two `\` in this file path. I don't think there is supposed to be a space between `:` and `target` here.